builtin/logical/rabbitmq/path_config_lease.go

```diff
@@ -0,0 +1,103 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigLease(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/lease",
+		Fields: map[string]*framework.FieldSchema{
+			"lease": &framework.FieldSchema{
+				Type:        framework.TypeString,
```

Comment:
Please use `framework.TypeDurationSecond` as it accepts both strings and integers and provides automatic conversion.